### ISSUE TITLE
QUESTION, Still being supported?

### ISSUE THREAD

**Issue by FelipeCardona95** [TorannD/ValheimLegends#3]
Hi Im planning to use this mod but with Mistlands update around the corner I was curious if you are planning to update as well when the times comes?

**Comment by vitharr137** [comment]
Agreed, love this mod's work and it would be amazing to have it in Mistlands
